lib/explorer/series.ex

```diff
@@ -27,7 +27,7 @@ defmodule Explorer.Series do
 
   @valid_dtypes Explorer.Shared.dtypes()
 
-  @type dtype :: :integer | :float | :boolean | :string | :date | :datetime
+  @type dtype :: :integer | :float | :boolean | :string | :date | :datetime | {:list, :integer}
```

Comment:
We probably want to make it recursive further on:

```suggestion
  @type dtype :: :integer | :float | :boolean | :string | :date | :datetime | {:list, dtype}
```